### ISSUE TITLE
How can i train on multiple graphics cards

### ISSUE THREAD

**Issue by scaactk** [jantic/DeOldify#296]
i have two RTX 3090 connected by nvlink，can i train model with bigger batch size?

**Comment by hdeping** [comment]
Maybe you need DataParallel (if you use pytorch), just see https://pytorch.org/docs/stable/generated/torch.nn.DataParallel.html

**Comment by jantic** [comment]
If you're just interested in getting results faster, DataParallel would work if you can get it to run with a GAN.  I tried briefly and realized it wasn't trivial.  And since then I haven't touched the subject because I haven't been using GANs for some time. 

But really I'd recommend trying to get DistributedParallel to work, along with incorporating SyncBatchNorm, so that you get the benefit of better batch norm statistics with bigger batch sizes. But there's definitely issues with the version of fastai used here when attempting to do DistributedDataParallel on GANs.  There was a patch for one of them [earlier this year](https://github.com/fastai/fastai/issues/2496).  That's not included in the version currently used by DeOldify. 

Also note that the newer version of fastai (quite different) appears to have better support for this.  That's another route- upgrade the code in a fork and try [this](https://docs.fast.ai/distributed.html).  

But I won't be adding support for this myself to the code base.  It's too much work that goes way beyond the scope of what I'm willing to do for free.
